README.md

```diff
@@ -57,12 +57,6 @@
 
```

Comment:
**[OPTIONAL]**

- [ ] Kindly remove the Tech stack which is not related to this project to make your readme descriptive, clean, and useful 😊